app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java

```diff
@@ -94,6 +94,11 @@
     public static final int SESSION_RELEASE_DISPLAY = 0;
     public static final int SESSION_DO_NOT_RELEASE_DISPLAY = 1;
 
+    @IntDef(value = { ACTIVATE, DEACTIVATE, DO_NOT_CHANGE})
+    public @interface SetSessionActiveState {}
+    public static final int ACTIVATE = 0;
```

Comment:
I don't see `ACTIVATE` used anywhere and I'm not sure I understand the use case.

Reply:
I guess I find it confusing because the flag is actually for the current session that becomes the old session and not the session being set. Maybe the flags should be `DEACTIVATE_CURRENT_SESSION` and `LEAVE_CURRENT_SESSION_ACTIVE`?

Reply:
I left that just in case we needed it in the future but I've removed it and now we just pass a flag, I guess if we need it sometime we can go back to the IntDef.